.github/workflows/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Check Markdown links
+
+on: 
+  push:
+  
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
this should probably be `actions/checkout@v3` instead - if they ever change their branch name this could break and in general it's best to pin to a specific version for reproducibility and handling explicit change

Reply:
@comses/comses-student-devs